packages/astro/src/@types/astro.ts

```diff
@@ -59,11 +59,15 @@ export interface AstroGlobal extends AstroGlobalPartial {
 }
 
 export interface AstroGlobalPartial {
-	fetchContent<T = any>(globStr: string): Promise<FetchContentResult<T>[]>;
 	/**
 	 * @deprecated since version 0.24. See the {@link https://astro.build/deprecated/resolve upgrade guide} for more details.
 	 */
 	resolve: (path: string) => string;
+	/** @deprecated Use `Astro.glob()` instead. */
+	fetchContent(globStr: string): Promise<any[]>;
+	glob(globStr: `${any}.md`): Promise<MarkdownInstance[]>;
```

Comment:
Any reason `MarkdownInstance` can't be a generic so you can type your `frontmatter` object? Not blocking, just curious.

Reply:
ah, tbh I misunderstood the types that you originally suggested and trimmed them down for clarity. Let me see about re-adding.

Reply:
done!

Reply:
Nice!